portfolio/src/main/webapp/script.js

```diff
@@ -62,3 +62,9 @@ function createCommentElement(comment){
   commentElement.appendChild(userCommentElement);
   return commentElement;
 }
+
+/** Tells the server to delete all comments. */
+function deleteComments() {
+  fetch('/delete-comments', {method: 'POST'});
+  window.location.reload();
```

Comment:
considering your post for /delete-comments returns a redirect to /index.html, you might not need to reload the page after calling /delete-comments. I would try commenting out this line and giving it a shot.

Reply:
When I had just the redirect the window didn't reload. I can take out the redirect without any consequences though, so I'll do that to avoid the redundancy

Reply:
interesting, sounds good!